src/client/components/pages/parts/collections-table.js

```diff
@@ -106,6 +106,7 @@ class CollectionsTable extends React.Component {
 									<th className="col-sm-2">Name</th>
 									<th className="col-sm-4">Description</th>
 									<th className="col-sm-2">Entity Type</th>
+									<th className="col-sm-2">No. of Entities</th>
```

Comment:
```suggestion
									<th className="col-sm-2">Total Entities</th>
```
a more better name for this column would be `Total Entities`

Reply:
I would even just say "Entities". I think in the context of this table it's self explanatory.